R/bmerge.R

```diff
@@ -90,6 +90,7 @@ bmerge = function(i, x, icols, xcols, roll, rollends, nomatch, mult, ops, verbos
           # we've always coerced to int and returned int, for convenience.
           if (verbose) cat("Coercing double column i.",names(i)[ic]," (which contains no fractions) to type integer to match type of x.",names(x)[xc],".\n",sep="")
           val = as.integer(i[[ic]])
+          attributes(val) = attributes(i[[ic]])
```

Comment:
is this something that should/could be left to `copyMostAttrib` at the C level?

At the R level, I think we usually have something like `if (!is.null(attributes(i[[ic]]))) attributes...`

Reply:
I think it should be very very trivial compared to the above use of as.integer().

Reply:
But if other codes add the if in front, I will adapt the convention. :D

Reply:
yep... only thought of it since i passed it recently in `CJ`